_field-types/supported-field-types/star-tree.md

```diff
@@ -0,0 +1,199 @@
+| `name` | Required | The name of the field. The field name should be present in the `properties` section as part of the index `mapping`. Ensure that the `doc_values` setting is `enabled` for any associated fields. |
+
+
+### Metrics
+
+Configure any metric fields on which you need to perform aggregations. `Metrics` are required as part of a star-tree configuration.
+
+When using `metrics`, follow these best practices: 
+
+- Currently, fields supported by `metrics` are all [numeric field types](https://opensearch.org/docs/latest/field-types/supported-field-types/numeric/), with the exception of `unsigned_long`. For more information, see [GitHub issue #15231](https://github.com/opensearch-project/OpenSearch/issues/15231). 
+- Supported metric aggregations include `Min`, `Max`, `Sum`, `Avg`, and `Value_count`. 
+    - `Avg` is a derived metric based on `Sum` and `Value_count` and is not indexed when a query is run. The remaining base metrics are indexed.
+- A maximum of `100` base metrics are supported per star-tree index.
+
+If `Min`, `Max`, `Sum`, and `Value_count` are defined as `metrics` for each field, then up to 25 such fields can be configured, as shown in the following example:
+
+```json
+{
+  "metrics": [
+    {
+      "name": "field1",
+      "stats": [
+        "sum",
+        "value_count",
+        "min",
+        "max"
+      ],
+      ...,
+      ...,
+      "name": "field25",
+      "stats": [
+        "sum",
+        "value_count",
+        "min",
+        "max"
+      ]
+    }
+  ]
+}
+```
+
+
+#### Properties
+
+The `metrics` parameter supports the following properties.
+
+| Parameter   | Required/Optional | Description  | 
+| :--- | :--- | :--- |
+| `name` | Required | The name of the field. The field name should be present in the `properties` section as part of the index `mapping`. Ensure that the `doc_values` setting is `enabled` for any associated fields. |
+| `stats` | Optional | A list of metric aggregations computed for each field. You can choose between `Min`, `Max`, `Sum`, `Avg`, and `Value Count`.<br/>Default is `Sum` and `Value_count`.<br/>`Avg` is a derived metric statistic that will automatically be supported in queries if `Sum` and `Value_Count` are present as part of metric `stats`.
+
+### Star-tree configuration parameters
+
+The following parameters are optional and cannot be modified following index creation.
+
+| Parameter  | Description   | 
+| :--- | :--- |
+| `max_leaf_docs` | The maximum number of star-tree documents that a leaf node can point to. After the maximum number of documents is reached, the nodes will be split based on the value of the next dimension. Default is `10000`. A lower value will use more storage but result in faster query performance. Inversely, a higher value will use less storage but result in slower query performance. For more information, see [Star-tree indexing structure]({{site.url}}{{site.baseurl}}/search-plugins/star-tree-index/#star-tree-index-structure).  |
```

Comment:
> the nodes will be split based on the value of the next dimension.

How about `once a node crosses threshold of `max_leaf_docs` , children nodes will be created based on the unique values` or something similar.